src/common/api/survey/survey.js

```diff
@@ -0,0 +1,15 @@
+import { request } from '../_request/request'
+
+/* API CALLS - Should return promises
+–––––––––––––––––––––––––––––––––––––––––––––––––– */
+export function sendSurvey(surveyObject, access_token) {
+  return request({
+    path: 'surveyResponse/',
+    data: {
+      access_token: access_token,
```

Comment:
nit-pick: reduce to `access_token,`

Reply:
Cool.  I will add a rule for this. Let's make a habit of adding a rule when a nit-pick comes up.